packages/react-core/src/components/TreeView/TreeView.tsx

```diff
@@ -45,6 +45,8 @@ export interface TreeViewProps {
   hasBadges?: boolean;
   /** Flag indicating if tree view has guide lines. */
   hasGuides?: boolean;
+  /** Flag indicating that tree nodes should be independently selectable, even when having children */
+  hasSelectableNodes?: boolean;
```

Comment:
Nitpick that I wouldn't block over: `isSelectable` could be an alternative prop name, as it would also align with our Card and Table selectable variants that have a similar prop.

Reply:
Yeah the prop name is something I wasn't too sure on. I thought about `isSelectable` but wasn't sure if it would be clear that it would make the nodes selectable opposed to the whole tree. I'm open to either prop name if anyone feels strongly one way or the other.

Reply:
Ah that's a fair point. I might think that having selectable items would make the tree view selectable in nature, but I do see how it isn't exactly clear. How do you feel about the idea of `TreeView` with prop `hasSelectableNodes`, and the `TreeViewListItem` with prop `isSelectable`?

Reply:
I like that, will update